modules/sdk-core/src/bitgo/inscription/iInscription.ts

```diff
@@ -0,0 +1,53 @@
+import { PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
```

Comment:
adding coin-specific libraries to `sdk-core` kind of defeats the purpose of modularizing the SDK